src/wazuh_modules/wmcom.c

```diff
@@ -85,18 +76,36 @@ size_t wmcom_getconfig(const char * section, char ** output) {
     return strlen(*output);
 }
 
-void wmcom_sync(const char * section, char ** output) {
-    const int ret = modulesSync(section);
+int wmcom_sync(char * buffer) {
+    const int ret = modulesSync(buffer);
     if(ret) {
-        mdebug1("At WMCOM sync: Could not sync '%s' section", section);
-        os_strdup("err Could not sync section", *output);
-    } else {
-        os_strdup("ok", *output);
+        mdebug1("At WMCOM sync: Could not sync '%s' buffer", buffer);
     }
-
+    return ret;
 }
 
 #ifndef WIN32
+
+void wcom_send(char * message)
+{
+    int sock;
+    if (sock = OS_ConnectUnixDomain(DEFAULTDIR WM_LOCAL_SOCK, SOCK_STREAM, OS_MAXSTR), sock < 0) {
```

Comment:
I'd suggest changing this, for a better understanding, as:
```
int sock = OS_ConnectUnixDomain(DEFAULTDIR WM_LOCAL_SOCK, SOCK_STREAM, OS_MAXSTR);
if (sock < 0)
{
...
}
```

Reply:
I agree with the comment, but because it is surrounded by legacy code with this style, it should look like this.